javaclient/src/main/java/confluo/rpc/RpcClient.java

```diff
@@ -115,6 +116,29 @@ public void setCurrentAtomicMultilog(String name) throws TException {
     curMultilogId = info.getId();
   }
 
+  /**
+   * get the atomic multilog id
+   *
+   * @param name The name of atomic multilog to set the current atomic multilog to
+   * @return  multilog id
+   * @throws TException Cannot get the atomic multilog
+   */
+  public long getAtomicMultilog(String name) throws TException {
```

Comment:
Perhaps rename this to getAtomicMultiLogId()

Reply:
renamed